pkg/beyla/os_nonlinux.go

```diff
@@ -9,3 +9,7 @@ func CheckOSSupport() error {
 func CheckOSCapabilities(_ *Config) error {
 	return nil
 }
+
+func KernelVersion() (major, minor int) {
+	return 5, 17
```

Comment:
should this now be 4, 18? Or does it even matter?

Reply:
Does not matter. It's just dummy stuff to let the linter/unit test compiler pass